lib/active_fedora/file.rb

```diff
@@ -139,6 +140,19 @@ def metadata?
       false
     end
 
+    def external_content?
+      external_url.present?
+    end
+
+    def external_url
+      return nil unless mime_type.start_with? "message/external-body"
+      mime_type[/url=['"](.+?)['"]/i, 1]
+    end
+
+    def external_url=(external_file_url)
+      self.mime_type = "message/external-body; access-type=URL; URL=\"#{external_file_url}\""
```

Comment:
That is currently how `File` and Fedora work.  I'm attempting to have `.content` return the bits for either external or internal.  Thus making the `File` act like it either has external or internal content but never both at the same time.  I had started with a separate `.external_content` method but thought this approach clearer.